tests/Integration/Behaviour/Features/Context/Domain/CurrencyFeatureContext.php

```diff
@@ -229,10 +254,10 @@ public function checkAPIData(TableNode $node)
 
             if ($expectedValue != $apiData[$key]) {
                 throw new RuntimeException(sprintf(
-                    'Invalid API data field %s: %s expected %s',
+                    'Invalid currency data field %s: %s expected %s',
                     $key,
-                    $apiData[$key],
-                    $expectedValue
+                    is_array($apiData[$key]) ? json_encode($apiData[$key]) : $apiData[$key],
```

Comment:
You can json_encode even if it's not an array :thinking:

Reply:
I know but I wanted to avoid the extra quotes Doesn't matter much  anyway I changed it